InternationalizationSample/InternationalizationSample/en.lproj/Localizable.strings

```diff
@@ -6,46 +6,6 @@
  Copyright (c) 2012 ForeSee. All rights reserved.
  */
 
-"FSPrivacyPolicyNavBarTitle"="Privacy Policy";
-"FSPrivacyPolicyLinkText"="Terms & Conditions";
-
-"FSInviteDeclineButtonText"="No, thanks";
-"FSInviteAcceptButtonText"="Yes, I'll help";
-"FSInviteTitle"="We want your feedback!";
-
-"FSContactInviteBody"="Can we email or text you later a brief customer satisfaction survey so we can improve your mobile experience?";
-"FSContactDetailsTitle"="Thank you for helping!";
-"FSContactDetailsBody"="Please provide your e-mail address or mobile number (US and CA only). After your visit you will be sent an e-mail/text message linking to the survey.";
-"FSContactDetailsDeclineButtonText"="Cancel";
-"FSContactDetailsHint"="email or mobile number";
-"FSContactDetailsEmptyInputError"="Enter a mobile number or email address";
-"FSContactDetailsInvalidInputError"="Format should be: name@domain.com or 123-456-7890";
-"FSContactDetailsAcceptButtonText"="Email/text me";
-"FSContactDetailsTextMessageNotificationButtonText"="Text me";
-"FSContactDetailsEmailNotificationButtonText"="Email me";
-
-"FSErrorDialogButtonText"="OK";
-"FSErrorDialogTitle"="Sorry";
-
-"FSExitInviteNotificationTitle"="You're invited!";
-"FSExitSurveyNotificationTitle"="Your survey has arrived!";
-
-"FSInSessionInviteBody"="You have been selected to participate in a brief customer satisfaction survey to let us know how we can improve your experience.";
-
-"FSLocalNotificationSurveyLinkBody"="As promised, here is your <ABC Company> survey. Thank you for your input!";
-"FSLocalNotificationInviteBody"="Can we contact you later with a brief customer satisfaction survey so we can improve your mobile experience?";
-
-"FSAttributionText"="Conducted by ForeSee";
-
-"FSSurveyNavBarTitle"="Customer Satisfaction Survey";
-"FSSurveyCloseButtonText"="Close";
-"FSSurveyLoadErrorMessage"="Due to a technical connection issue we are unable to provide you with a survey at this time. Thank you for your time and interest.";
-
-"FSCouldNotLoadSurveyTitle"="Survey Load Fail";
-"FSCouldNotLoadSurveyMessage"="Couldn't load the survey at this time. There could be a problem with your connectivity.";
-
-"FSWebCloseButtonText"="Done";
-"FSWebLoadErrorTitle"="Page Load Fail";
-"FSWebLoadURLNotWhitelistedErrorMessage"="Couldn't load page. Only whitelisted domains are permitted.";
-"FSWebLoadErrorMessage"="Couldn't load page at this time. There could be a problem with your connectivity.";
-"FSWebLoadErrorButtonText"="OK";
+"FORESEE_Invite_InSession_DeclineButton"="No, thanks";
+"FORESEE_Invite_InSession_AcceptButton"="Yes, I'll help";
+"FORESEE_Invite_InSession_InvitePage_Title"="We want your feedback!";
```

Comment:
I wonder why overwriting the strings here in the sample app? Aren't they already handled in the SDK (and are same strings)?

Reply:
It's just to show the user how they can override them, if they want to